### PR TITLE
Vampire stuns last longer and now mute

### PR DESCRIPTION
**Pull request by thefastfoodguy**
🆑
balance: vampire stuns last slightly longer
balance: Vampire stuns now mute.
/🆑

goes well with #7462

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
Why do you wish to change vampire stuns in such a way?

**Comment by thefastfoodguy** [comment]
Because all it takes is one fast-typer yelling for help over comms and then SUDDEN LYNCH MOB IN MAINT.

**Comment by Anoonki** [comment]
I can agree with this being a thing, because right now Vamps are really weak to start, especially if #7462 goes through.
This won't change what monsters vampires become later, but will make their early game less painful to try and push through without getting valid'd.

**Comment by scrubmcnoob** [comment]
Vampires are weak early game but are by far the hardest antag to fight if said player knows how to use all their abilities once they unlock them.
I really don't think their glare needs to be mute.

File: code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -222,10 +222,11 @@
 	for(var/mob/living/target in targets)
 		if(!affects(target))
 			continue
-		target.Stun(5)
-		target.Weaken(5)
+		target.Stun(7)
+		target.Weaken(7)
+		M.AdjustSilence(10)
```
Comment (Citinited) [inline code/game/gamemodes/vampire/vampire_powers.dm]: Think `M` needs changing to `target` here. Should fix up Travis error

Comment (SamHPurp) [inline code/game/gamemodes/vampire/vampire_powers.dm]: ^

You should at least be testing your code before submitting it.

**Comment by SamHPurp** [comment]
> balance: vampire stuns last slightly longer
> balance: Vampire stuns now mute.

it should be tweak: and add: respectively.

-------

I don't think I agree with this...  a suggestion for something similar though, when Hypnotize wears off, they could get a message like convertees from the Missionary Staff do.

**Comment by Fox-McCloud** [comment]
14 second stun and 20 second silence.

This pretty much turning vamps into the worst of old changelings; para sting, silence sting, nothing to be done about it.

You get glared, you're dead, literally every single time, because it's more than enough time to not only cuff them, and remove their headset, but also relocate them.

**Comment by deep-fried-faceplant** [comment]
Vampires need a buff, but this is not how to do it.

**Comment by Necaladun** [comment]
This would be incredibly frustrating. If you get glared alone, you're dead. It would make things just too easy for the vampire to take someone out, and no fun at all for the person getting hit. Glare is already quite powerful as it is.

**Comment by Ralta** [comment]
Orrrrrrr you could do what I and other vampires do and prep like you would with other antags. I always make a stunprod and cable cuffs as a vampire and then make a little hidey-hole through a hidden wall in maint. Vampires are supposed to be weak to start; I don't think they really need another buff at the beginning...
-1 from me.

**Comment by Birdtalon** [comment]
Poor balance decision which would make their starting ability one of their most powerful with no counter. 👎

**Comment by tigercat2000** [comment]
After discussion, it was decided to close this PR; while vampires may need tweaks and potential buffs, turning their basic, free, spammable glare into another version of a Shadowling Glare is not the way to go, and, as Fox pointed out, merely emulates the worst aspects of the old Changeling stun sting.

**Pull request closed without merging**